packages/client/src/core/contexts/ThemeContext.tsx

```diff
@@ -41,22 +43,96 @@ export const useTheme = () => {
 export const ThemeProvider: React.FC<
   PropsWithChildren
 > = ({ children }) => {
+  // Сначала пытаемся получить тему из localStorage, если её там нет - по умолчанию 'light'
+  const themeStart =
+    typeof window !== 'undefined' &&
+    localStorage.getItem('theme')
```

Comment:
_Не критично, но лучше исправить_

По возможности желательно всегда выносить повторяющиеся хардкодные константы. Это поможет в очередной момент не опечататься и не допустить ошибку

```suggestion
    localStorage.getItem(THEME_KEY)
```

Reply:
сделаю